### DISCUSSION TITLE
Looks promising, but find the file list and search taek too much spage.

### DISCUSSION THREAD

**Discussion by russetrob** [community/community#48932] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

The functionality looks promising I can search in C++ :-)
But the layout eats too much white space, while not allowing the code view to grow to absorb the white space on a large screen.
End up with file list at far edge of screen, a skinny code view in middle of screen, and search box to right of that.

Maybe accept that if searching for something, want to see as much code on screen as possible, at expense of clean layout.

**Comment by look** [comment, marked as answer]
@russetrob thanks for the feedback. You can choose a wider layout by clicking the "..." menu while viewing a file and unchecking "Center content"

<img width="276" alt="Screen Shot 2023-03-07 at 3 02 36 PM" src="https://user-images.githubusercontent.com/10186/223574091-476fb947-6203-4d23-ba1a-3ae3ad2f9492.png">
